src/server.js

```diff
@@ -23,12 +29,143 @@ const sendUserError = (err, res) => {
   }
 };
 
+const restrictedAccess = (req, res, next) => {
+  const path = req.path;
+  if (path.includes('/restricted')) {
+    if (req.session.name) {
+      User.findOne({ username: req.session.name })
+        .then((user) => {
+          req.user = user;
```

Comment:
Nice job wrapping all of the single parameters in parens when followed by the curly braces.